x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_columns/use_columns.test.tsx

```diff
@@ -296,7 +346,9 @@ describe('useColumns', () => {
         { wrapper }
       );
 
-      await waitFor(() => expect(result.current.columns).toMatchObject(defaultColumns));
+      act(() => {
+        expect(result.current.columns).toMatchObject(resultColumns);
```

Comment:
If I'm not mistaken, the `act` function should only be used to clear pending updates before making assertions

Reply:
You are right. Fixed it.